test/unit/increment.test.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var chai = require('chai')
+  , expect = chai.expect
+  , Support   = require(__dirname + '/../support')
+  , current   = Support.sequelize
+  , Sequelize = Support.Sequelize
+  , sinon     = require('sinon');
+
+describe(Support.getTestDialectTeaser('Model'), function() {
+  describe('increment', function () {
+    describe('options tests', function() {
+      var stub
+        , Model = current.define('User', {
+          id: {
+            type:          Sequelize.BIGINT,
+            primaryKey:    true,
+            autoIncrement: true,
+          },
+          count: Sequelize.BIGINT,
+        });
+
+      before(function() {
+        stub = sinon.stub(current, 'query').returns(
+          Sequelize.Promise.resolve({
+            _previousDataValues: {id: 1},
+            dataValues: {id: 3}
+          })
+        );
+      });
+
+      after(function() {
+        stub.restore();
+      });
+
+      it('should allow increments even if options are not given', function () {
```

Comment:
We should require `where` to be defined, like we do for update, so users don't accidentally update all rows in the database